src/solver/optimisation/adequacy_patch.cpp

```diff
@@ -95,28 +106,24 @@ void setBoundsAdqPatch(double& Xmax,
                        const int Interco,
                        PROBLEME_HEBDO* ProblemeHebdo)
 {
-    AdequacyPatch::LinkCapacityForAdequacyPatchFirstStep
-      SetToZeroLinkNTCForAdequacyPatchFirstStep;
+    LinkCapacityForAdequacyPatchFirstStep SetToZeroLinkNTCForAdequacyPatchFirstStep;
 
     SetToZeroLinkNTCForAdequacyPatchFirstStep
-      = SetNTCForAdequacyFirstStep(
-        ProblemeHebdo->StartAreaAdequacyPatchType[Interco],
-        ProblemeHebdo->EndAreaAdequacyPatchType[Interco],
-        ProblemeHebdo->adqPatch->AdqBehaviorMap);
+      = SetNTCForAdequacyFirstStep(ProblemeHebdo->StartAreaAdequacyPatchType[Interco],
+                                   ProblemeHebdo->EndAreaAdequacyPatchType[Interco],
```

Comment:
there will be conflict with PR adq run time data, just remember to resolve in this PR or that PR

Reply:
Yes. I also realized that. Will solve it! Thanks.